docs/learn/learn-guides-claims.md

```diff
@@ -0,0 +1,111 @@
+---
+id: learn-guides-claims
+title: Polkadot-JS Guides about Claims
+sidebar_label: Claim Guides
+description: Step-by-step guide on how to claim your tokens.
+keywords: [claims, tokens, kusama, polkadot]
+slug: ../learn-guides-claims
+---
+
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
+
+<MessageBox message="Polkadot-JS is for developers and power users only. If you need help using the
+[Polkadot-JS UI](../general/polkadotjs-ui.md), you can contact the
+[Polkadot Support Team](https://support.polkadot.network/support/home). For more user-friendly tools
+see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashboards-index) pages." />
+
+<DocCardList />
+
+If you participated in a previous DOT sale before 2020 and received your DOT allocation indicator
+tokens, you can now claim your DOT (and a proportional amount of KSM on the Kusama network). The
+claim process connects the address where your DOT indicators have been stored on Ethereum with a
+native {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} address and, if your ETH
+address is eligible, will pay the tokens to the
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} address.
+
+To do this, you must sign a message on Ethereum containing the address of your
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account. You can do this by using the
+Polkadot-JS UI [Claims app](https://polkadot.js.org/apps/#/claims). Ensure that you are connected to
+the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network (displayed in the
+upper-left-hand corner of the screen).
+
+:::warning Third-party claim processes
+
+Claiming using a third-party process can lead to losing your allocation; therefore, we cannot
+recommend using any third-party apps. Manually specifying your transaction data, as specified in our
+claims process below, is the only way to ensure you will receive your allocation.
+
+:::
+
+## Generate an Account
+
+You will need to generate a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account
+to claim {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. See the
+[available wallets and extensions](../general/wallets-and-extensions.md) for more information about
+wallets and browser extensions you can use to create an account. In terms of hardware wallet
+support, you can use the [Ledger](../general/ledger.md) devices or
+[Polkadot Vault](../general/polkadot-vault.md).
+
+### Address Conversion
+
+If you need to (most likely you don't), you can use
+[this tool](https://www.shawntabrizi.com/substrate-js-utilities/) to convert the Substrate SS58
+Address to the hexadecimal public key.
+
+![SS58 to Hex](../assets/ss58-to-hex.png)
+
+## Claiming Tokens
+
+The Polkadot-JS [Claims app](https://polkadot.js.org/apps/#/claims) will guide you through the
+process of claiming the tokens.
+
+### Select Account on Polkadot
```

Comment:
You can remove Polkadot from the section title to remove ambiguity, as this doc is applicable for both Polkadot and Kusama.

Reply:
Also, it can be confusing to see Polkadot screenshots while claiming KSM

Reply:
will add an info banner on the top of the page for that